src/boot/vue-simple-acl.js

```diff
@@ -4,21 +4,26 @@ import { computed } from 'vue';
 import { useUserStore } from 'stores/UserStore';
 
 const userStore = useUserStore();
-const userRoles = computed(() => ({ roles: userStore.roles }));
+const userPermissions = computed(() => ({ permissions: userStore.permissions }));
+
+const findPermission = (user, action, entity) => user.permissions.find(
+  ({ action: userAction, entity: userEntity }) => userAction === action && userEntity === entity,
+);
+
 const rules = () => defineAclRules((setRule) => {
-  setRule('admin', (user) => process.env.HAS_BACKEND && process.env.ADMIN_URL && user.roles.includes('admin'));
-  setRule('create-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagram'));
-  setRule('create-diagram-from-template', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagramFromTemplate'));
-  setRule('create-component', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createComponent'));
-  setRule('create-component-from-template', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createComponentFromTemplate'));
-  setRule('create-project', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createProject'));
-  setRule('create-project-from-template', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createProjectFromTemplate'));
-  setRule('delete-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_deleteDiagram'));
+  setRule('admin', (user) => process.env.HAS_BACKEND && process.env.ADMIN_URL && findPermission(user, 'ACCESS', 'ADMIN'));
```

Comment:
I think you can remove `proccess.env.ADMIN_URL` in all application (don't forget the config/Readme)
Because now, we have an endpoint on api to redirect on Admin app: `/api/redirect?app=admin`
So, now Leto-modelizer don't need to know the url of admin app, because it have to go on `/api/redirect?app=admin`

Reply:
done